.travis.yml

```diff
@@ -3,6 +3,9 @@ language: python
 python:
   - "2.7"
 
+notifications:
+  email: false
```

Comment:
rather than doing this i feel like you should do this on a profile level, if possible. Since this isn't a standard in our travis.yml files, this'll confuse people in the future.